### ISSUE TITLE
Branch for nim 0.19 prereleases (nim-git in AUR)

### ISSUE THREAD

**Issue by stefanhusmann** [kitsunyan/pakku#36]
Compiling pakku using nim-git currently fails. Merely because "future" module was renamed to "sugar" and "SystemError" was replaced by "CatchableError".

I therefore suggest to have a branch targeting to upcoming nim version, if you do not have one already.

I have this patch, but the build still fails. 

NIM: src/pakku
utils.nim(346, 29) Error: type mismatch: got <char>
but expected one of: 
proc toHex(x: BiggestInt; len: Positive): string
proc toHex[T: SomeInteger](x: T): string
proc toHex(s: string): string

expression: toHex
make: *** [Makefile:123: src/pakku] Error 1

**Comment by juancarlospaco** [comment]
Compile Fails with current Nim Stable, same error. No one can install Pakku. :sob:

**Comment by kitsunyan** [comment]
@juancarlospaco that's not true. New nim stable is available only in community-testing repo. Moreover, this package is broken and shoudn't be moved to community until the issue will be addressed: https://bugs.archlinux.org/task/60490 (although there is workaround).

There are more fixes I should do for 0.19 support. Right now I'm working on this: https://github.com/nim-lang/Nim/commit/85ea9593b38351e69fedac61ff0c2b958bac4b7f and this problem is much more severe and complex than just replacing SystemError with CatchableError.
